CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ else(MSVC)
       CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7.0)
     set(CMAKE_CXX_FLAGS "-faligned-new ${CMAKE_CXX_FLAGS}")
   endif()
-  include(cmake/clang_flags.cmake)
+  include(cmake/modules/clang_flags.cmake)
```

Comment:
Please use ClangFlags.cmake (CamelCase) to be consistent with the rest of the files under modules